src/frontend/js/components/DashboardItem/DashboardItemOrder.stories.tsx

```diff
@@ -0,0 +1,56 @@
+import { ComponentMeta, ComponentStory } from '@storybook/react';
+import { DashboardItemOrder } from './DashboardItemOrder';
+import { Order, OrderState } from '../../types/Joanie';
+import { enrollment } from './mock.stories';
+
+const order: Order = {
```

Comment:
Could we use `OrderFactory` here ?

Reply:
I think so, planned to do that after this first overall review :)